### ISSUE TITLE
deepar_synthetic.ipynb issues

### ISSUE THREAD

**Issue by aaronmarkham** [awslabs/amazon-sagemaker-examples#1353]
https://github.com/awslabs/amazon-sagemaker-examples/blob/master/introduction_to_amazon_algorithms/deepar_synthetic/deepar_synthetic.ipynb

1. installs s3fs, but this is already available, plus it generates a lot of warnings and slows down running the notebook.
1. doesn't use a default s3 bucket
1. error
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-9-ee6bcf6b48fa> in <module>
      8     noise = sig*np.random.randn(data_length)
      9     data = source + noise
---> 10     index = pd.DatetimeIndex(start=t0, freq=freq, periods=data_length)
     11     time_series.append(pd.Series(data=data, index=index))

TypeError: __new__() got an unexpected keyword argument 'start'
```

**Comment by aaronmarkham** [comment]
https://github.com/awslabs/amazon-sagemaker-examples/pull/1302 may have a fix for this.

**Comment by aaronmarkham** [comment]
https://github.com/awslabs/amazon-sagemaker-examples/pull/1219 also seems to address the error.

**Comment by aaronmarkham** [comment]
https://github.com/awslabs/amazon-sagemaker-examples/pull/1273 also addresses the error.

**Comment by aaronmarkham** [comment]
The 3rd issue has been resolved by #1219 ... leaving open to track the other two issues.
